sources/bash-helpers.sh

```diff
@@ -3,5 +3,7 @@
 function tag_timestamp {
   # This timestamp format can be lexicographically sorted
   # which is important for flux automation.
-  echo $(date +%Y.%m.%d.%H.%M.%S)
+  # This is in UTC so that timestamps will be the same,
+  # whether this script is run directly from a developer's machine or from a github action
+  echo $(date -u +%Y.%m.%d.%H.%M.%S)
```

Comment:
whaaaaaaat? I was expecting [this line in fingerprinter's `build.sh` to be updated](https://github.com/UWIT-IAM/fingerprinter/blob/main/fingerprinter/build.sh#L233)

I wonder how many separate + equivalent build pipeline pieces we have

Reply:
Yup, this is the one, at least from what I can see. The `tag_timestamp` function (which is what updated) is then used at:
https://github.com/UWIT-IAM/identity-uw/blob/develop/scripts/deploy.sh#L123